tests/ops/service/pagination/test_pagination_strategy_link.py

```diff
@@ -49,6 +49,42 @@ def response_with_empty_string_link():
     return response
 
 
+@pytest.fixture(scope="function")
+def response_with_has_next_conditional_true():
+    response = Response()
+    response._content = bytes(
+        json.dumps(
+            {
+                "customers": [{"id": 1}, {"id": 2}, {"id": 3}],
+                "links": {
+                    "next": "https://domain.com/customers?page=def",
+                    "hasNext": "true",
```

Comment:
Now that we know what the actual payload looks like, we should replace this with a boolean
```suggestion
                    "hasNext": True,
```
Same thing for the other fixture

Reply:
Replacing them with base boolean. 

I was thinking, would it be redundant to also test that it keeps catching these string values correctly? I.E if a API returns TRUE/FALSE in all caps, or something similar

Reply:
I think this is ok for now, we can handle those use cases if we run into them